src/tfr_visualization/config/mapping.rviz

```diff
@@ -66,11 +65,11 @@ Visualization Manager:
         base_footprint:
           Value: true
         base_link:
-          Value: true
+          value: true
         duo3d/camera_frame:
-          Value: true
+          value: true
         duo3d/frame:
-          Value: true
+          value: true
         duo3d/imu_frame:
           Value: true
```

Comment:
Should the other `Value` variables be changed to lowercase `value`?

Reply:
This is generated code, so I can't really change it, however in our system for our yaml, yes you are right.